src/submission/submission-to-clinical/merge-submission.ts

```diff
@@ -76,15 +176,29 @@ const addOrUpdateClinicalEntity = (
 };
 
 /* ********************************* *
- * Some repeated convenience methods *
+ * Convenient private methods        *
```

Comment:
I think it makes sense if these are just private methods used in merge-submission only, because that way we can change how the merging happens and not worry about how we get clinical entites from the donor